lang/de_DE.trn

```diff
@@ -2054,20 +2054,20 @@ t954 Kopfdaten ändern
 o955 Found a previous fixed installation
 t955 Eine vorherige Installation wurde gefunden
 o956 Left-Click and drag to the left or the right
-t956 
+t956 Linksklick und nach links oder nach rechts ziehen
 o957 Cancel Command Block Offset
-t957 
+t957 Befehlsblockversetzung abbrechen
 o958 Cancels the command blocks coords changed when copied.
-t958 
+t958 Bricht die Versetzung der Koordinaten in den Befehlsblöcken beim Kopieren ab.
 o959 Weird stuff happen!
-t959 
+t959 Komische Dinge passieren!
 o960 Enable Custom Name
-t960 
+t960 Aktiviere Name
 o961 Select a Minecraft level....
-t961 
+t961 Wähle eine Minecraft Welt....
 o962 Levels and Schematics
-t962 
+t962 Welten und Schematics
 o963 Show Block Info when hovering
-t963 
+t963 Zeige Block Informationen beim drüberfahren
```

Comment:
Honestly, I can't tell what form shall be used.
I the French translation, I use abbreviations to shorten some widget texts. I even strip off some words: 'Camera Acceleration' is translated 'Accélération caméra' instead of 'Accélération *de la* caméra'.
But when translating tooltip or other pieces of text, I generally don't shorten them.

The string 963 is a widget text, and the 964 is the related tooltip.